run.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+ZK_URL="${1:-zk://127.0.0.1:2181/metronome}"
+MESOS_MASTER_URL="${2:-127.0.0.1:5050}"
+PLAY_PORT="${3:-8888}"
```

Comment:
why don't we call it `HTTP_PORT`? ;) And it `8888` or `8989` a better default one?

Reply:
good point, thanks

Reply:
agree with zen-dog and think the default should be 9000 like it is every where else